tests/cross/src/test/scala/tests/pc/CompletionSnippetSuite.scala

```diff
@@ -103,7 +103,12 @@ class CompletionSnippetSuite extends BaseCompletionSuite {
       |}
       |""".stripMargin,
     """|MyType
-       |""".stripMargin
+       |""".stripMargin,
+    compat = Map(
+      "2.11" -> """|MyType
+                   |MTOM
```

Comment:
Any idea, where is this coming from?

Reply:
Classpath:
```
javax/xml/ws/soap/MTOM.class
javax/xml/ws/soap/MTOMFeature.class
```
Seems to be a Java 8 thing.

Reply:
Thanks for checking!